content/en/agent/basic_agent_usage/_index.md

```diff
@@ -148,6 +148,7 @@ When the Agent is running, use the `datadog-agent launch-gui` command to open th
 | [macOS][9]                                      | macOS 10.12+                                      |
 | [Windows server 64-bit][10]                     | Windows Server 2008r2+ and Server Core (not Nano) |
 | [Windows 64-bit][10]                            | Windows 7+                                        |
+| [Windows Azure Stack HCI OS][10]                | All Versions                                      |
```

Comment:
The link is pointing to: https://docs.datadoghq.com/agent/basic_agent_usage/windows/

Is that good?

Reply:
Yes- the process is exactly the same as the instructions there indiciate. The only difference is that there is no GUI option, but the same cli/powershell commands that are already provided there work for AzSCHI. I don't think there's a need to call that out specifically anywhere though.